src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vedtak/vedtaksperiode/Opphørsperiode.kt

```diff
@@ -115,6 +122,44 @@ fun slåSammenOpphørsperioder(alleOpphørsperioder: List<Opphørsperiode>): Lis
     }
 }
 
+private fun finnOpphørsperioderPåGrunnAvReduksjonIRevurdering(
+    forrigeUtbetalingsperioder: List<Utbetalingsperiode>,
+    utbetalingsperioder: List<Utbetalingsperiode>,
+): List<Opphørsperiode> {
+    val erUtbetalingOpphørtTidslinje =
```

Comment:
Kanskje bare dropper `er` prefixen? Leser litt som en `Boolean` variabel slik den er navngitt nå 😅 

```suggestion
    val utbetalingOpphørtTidslinje =
```

Reply:
Hva å snu på det, OpphørtUtbetalingTidslinje? :P